tabular/src/autogluon/tabular/models/catboost/catboost_model.py

```diff
@@ -53,9 +53,60 @@ def _preprocess_nonadaptive(self, X, **kwargs):
                     X[category] = X[category].cat.add_categories('__NaN__').fillna('__NaN__')
         return X
 
+    #TODO: could be useful if we could still exploit the early stopping feature during incremental training
```

Comment:
Why is it not compatible? I already do this when time_limits is specified by training a small amount of iterations, then incrementally training via warm start the rest which uses early stopping in both cases.